README.md

```diff
@@ -2,8 +2,52 @@
 
 ## Purpose
 
-_... help fill this in!_
+Purpose is *not* to respond to reports of security issues with Node.js, that is
+done by the "security team", see https://github.com/nodejs/node#security
+
+*todo*: should the team be identified in
+https://github.com/nodejs/node#security?
```

Comment:
I think I'd leave a decision on that until the group is up and running and we have a better feel as to what it is doing day to day.